chart/env/prod.yaml

```diff
@@ -197,6 +197,22 @@ workers:
         cpu: 2
         memory: "8Gi"
     tolerations: []
+  -
+    deployName: "light"
+    maxJobsPerNamespace: 20
+    workerOnlyJobTypes: "dataset-split-names-from-streaming,dataset-split-names-from-dataset-info"
```

Comment:
this worker could contain all the "light" jobs (parquet, dataset-info, sizes, etc)

Reply:
Done in commit https://github.com/huggingface/datasets-server/pull/945/commits/078b4a2b77796683656f829fbfc9ba0c8eaa61e1